spec/models/organization_spec.rb

```diff
@@ -48,4 +48,16 @@
       expect(organization.provider?).to be true
     end
   end
+
+  describe '#providers' do
+    it 'includes providers in scope' do
+      org = described_class.create!(provider: true)
+      expect(described_class.providers).to include(org)
+    end
+
+    it 'excludes non-providers in scope' do
+      org = described_class.create!(provider: false)
+      expect(described_class.providers).not_to include(org)
+    end
+  end
```

Comment:
this is sort of what i was alluding to in standup — if you define a super simple scope using ActiveRecord on one field, there isn't really a way for it to fail unless Rails's own code fails, so you're effectively testing Rails. not a bad thing necessarily, just generally less useful than other kinds of tests

Reply:
@thatbudakguy fair! I couldn't think of another way to test it.

Reply:
also - destroy permissions!!!